final_task/rss_reader/requirements.txt

```diff
@@ -0,0 +1,28 @@
+arguments==76
+beautifulsoup4==4.8.1
+bs4==0.0.1
+certifi==2019.9.11
+cffi==1.13.2
+chardet==3.0.4
+colorama==0.4.1
+colorize==1.1.0
+cycler==0.10.0
+dominate==2.4.0
+entrypoints==0.3
+feedparser==5.2.1
+flake8==3.7.9
+fpdf==1.7.2
+idna==2.8
+kiwisolver==1.1.0
+lxml==4.4.1
+mccabe==0.6.1
+numpy==1.17.4
+pycodestyle==2.5.0
+pycparser==2.19
+pyflakes==2.1.1
+pyparsing==2.4.5
+python-dateutil==2.8.1
+requests==2.22.0
+six==1.13.0
+soupsieve==1.9.5
+urllib3==1.25.7

```

Comment:
There are too many redundant libs here. Keep in mind that you should specify only libs that are _directly_ required by your application.